.circleci/config.yml

```diff
@@ -778,10 +717,6 @@ jobs:
           name: Clippy linting on vm (no feature flags)
           working_directory: ~/project/packages/vm
           command: cargo clippy -- -D warnings
-      - run:
-          name: Clippy linting on vm (all feature flags)
-          working_directory: ~/project/packages/vm
-          command: cargo clippy --features cranelift,iterator -- -D warnings
```

Comment:
Let's leave this "all feature flags" version in here, but just remove the cranelift feature. But we should update the two VM lints anyways:

no feature flags: `cargo clippy --no-default-features -- -D warnings`
all feature flags: `cargo clippy --features iterator,staking,stargate -- -D warnings`